introduction.md

```diff
@@ -1,3 +1,8 @@
 Hello everyone.  This is my formal introduction to the dojo. My name is Chad, username: chadjguidry.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?